lib/index.js

```diff
@@ -54,14 +54,41 @@ const getAllTests = flatMap(suite => [
   ...getAllTests(suite.suites),
 ])
 
+const getStateTimeSpan = reports => {
+  const spans = reports.map(({ stats: { start, end } }) => {
+    return { start: new Date(start), end: new Date(end) }
+  })
+
+  const maxSpan = spans.reduce(
+    (currentMaxSpan, span) => {
+      const start = new Date(
+        Math.min(currentMaxSpan.start.getTime(), span.start.getTime())
+      )
+      const end = new Date(
+        Math.max(currentMaxSpan.end.getTime(), span.end.getTime())
+      )
+      return { start, end }
+    },
+    { ...spans[0] }
```

Comment:
Why do you need a spread here? I think, here it’s possible to reduce without supplying the initial value at all

Reply:
yes, it can be removed